pkg/services/sqlstore/dashboard.go

```diff
@@ -104,6 +104,8 @@ func (ss *SQLStore) FindDashboards(ctx context.Context, query *search.FindPersis
 		filters = append(filters, filter)
 	}
 
+	filters = append(filters, query.Filters...)
```

Comment:
would you like to filter only with dashboard.title as in the unittest? if it is the case, you could use the field query.Title, it would be built into query.Title Like "%xxx%".

Reply:
If you would like to use the query.Filters on other fields, the PR looks good.

Reply:
I plan to use filters for different fields. I used title in unit-tests just for the sake of the example